docs/README.md

```diff
@@ -0,0 +1,76 @@
+## pyIncore_incubator documentation
+
+**pyIncore_incubator** documentation which is built using Python [Sphinx](http://www.sphinx-doc.org/en/master/) package.
+
+### Installation
+
+Clone the code from pyincore repository [git](https://github.com/IN-CORE/pyincore-incubator.git) 
```

Comment:
Clone the code from `pyincore-incubator`?

Reply:
fixed